lib/OpenAPI/Client.pm

```diff
@@ -11,8 +11,6 @@ use constant DEBUG => $ENV{OPENAPI_CLIENT_DEBUG} || 0;
 
 our $VERSION = '1.03';
 
-my $BASE = __PACKAGE__;
```

Comment:
I can't remember why I used $BASE and not $class (or $parent). I have to look at the git log too see if I added a comment about it...